sui_programmability/examples/nft_standard/README.md

```diff
@@ -0,0 +1,115 @@
+# NFT Standards, Philosophy, and Requirements
+
+# Basics
+
+Sui’s object-centric data model gives Sui objects many of the features of a basic ERC721-style NFT: an owner, the ability to be transferred, a globally unique identifier.
+
+### Individual NFT’s
+
+An NFT is a Sui object (i.e., a struct value whose declared type has the `key` ability, which in turn means the struct value has a field named `id` holding a globally unique ID). All NFT’s are Sui objects, but not every Sui object is an NFT.
```

Comment:
On the Sui network:
Token is Object.
NFTs are Objects.
Make a general trading market for Objects, including token defi and NFT defi.
Object is valuable because Storage Fund supports the bottom price.